backend/src/graphql/resolver/GdtResolver.ts

```diff
@@ -25,13 +25,17 @@ export class GdtResolver {
     const userRepository = getCustomRepository(UserRepository)
     const userEntity = await userRepository.findByPubkeyHex(context.pubKey)
 
-    const resultGDT = await apiGet(
-      `${CONFIG.GDT_API_URL}/GdtEntries/listPerEmailApi/${userEntity.email}/${currentPage}/${pageSize}/${order}`,
-    )
-    if (!resultGDT.success) {
-      throw new Error(resultGDT.data)
+    try {
+      const resultGDT = await apiGet(
+        `${CONFIG.GDT_API_URL}/GdtEntries/listPerEmailApi/${userEntity.email}/${currentPage}/${pageSize}/${order}`,
+      )
+      if (!resultGDT.success) {
+        throw new Error(resultGDT.data)
+      }
+      return new GdtEntryList(resultGDT.data)
+    } catch (err: any) {
+      throw new Error(err)
```

Comment:
So this will keep throwing?! 

I really would like a frontend implementation here telling the user something like `Due to an error we currently cannot show you your GDT` or alike in the GDT list to inform the users about whats happening.